dz12_ex2.py

```diff
@@ -0,0 +1,24 @@
+# 2. Написати декоратор, який буде записувати в файл назву функції, яку він декорує, і писати час її виклику.
+
+import time
+
+def decor(func):
+    '''декоратор виконує друк назви функції і часу, коли вона була викликана.'''
+    def wrap(*args, **kwargs):
+        func(*args, **kwargs)
```

Comment:
assign the result of the `func` call to some variable and the return this variable after you write to a file (but outside of the context manager block) to be able to decorate function that return some value